src/Stratis.Bitcoin.Features.Consensus/Extensions.cs

```diff
@@ -18,5 +21,40 @@ public static class Extensions
 
             return await Task.FromResult(true);
         }
+
+        public static long GetSizeInBytes(this UnspentOutputs unspentOutputs)
```

Comment:
This is a very unfortunate placement of such methods. Just imagine that serialisation methods would be implemented here with extensions. It completely disconnects from the object itself and that object is changed you will very likely change the serialisation method because you see it in the class in the same file, but I can guarantee you that you will never find this one so you will not change it.

Reply:
Moved each one of those to a corresponding object